### ISSUE TITLE
Render stream names at z14 like other waterway names

### ISSUE THREAD

**Issue by Adamant36** [gravitystorm/openstreetmap-carto#3114]
It looks like canal and river names render at z14 while stream names currently don't. It would be good if they all rendered at the same level.

**Comment by matkoniecz** [comment]
Can you give example of place where it would improve rendering?

> It would be good if they all rendered at the same level.

Given that stream are of lower importance than rivers current situation is not surprising for me.

**Comment by Adamant36** [comment]
@matkoniecz In northern california there are a lot of major waterways that are notable which are called creeks, but are larger then streams and are to small to be labeled as rivers. Id put them in the same  category as canals. There currently isnt a way to tell them apart from smaller unamed streams on the map though and a lot of were wrongly tagged as rivers. Which is why I think the name should be rendered at z14 like canals are so they can seen at that zoom level without having to miss tag them. I think it would help when planning hiking routes or something at higher zoom levels to if the name was shown. 

Heres a few examples
http://www.openstreetmap.org/#map=14/40.5936/-122.3417
Churn Creek is a major local creek of historical significence but its not easy to seperate it from the other more minor streams unless your zoomed in so far it doesnt even matter anymore. 

http://www.openstreetmap.org/#map=15/39.8213/-122.0138
Theres also things like this place where you cant tell where pine creek branches out from or goes to compared the other unamed streams unless your zoomed in to close for it to matter. 

Theres a lot of similar examples in the same area.

**Comment by boothym** [comment]
>but are larger then streams and are to small to be labeled as rivers. Id put them in the same category as canals. 

In OSM there's nothing in between, so if it's bigger than a stream - i.e. it can be jumped across - then it's a river. A canal is an artificial version of a river. See https://wiki.openstreetmap.org/wiki/Tag:waterway%3Dstream

You changed https://www.openstreetmap.org/way/91683713/ from a river to a stream - however it looks wide enough to be a river, especially south of Old Alturas Road.

**Comment by Adamant36** [comment]
@boothym I'm aware of that. I was going to mention it, but I was trying to keep it from divulging into a tagging discussion. 

I think the argument could made though that the whole "being able to jump across" isn't really a good way to define something like a water way because it assumes consistently small level of flow. If you look at the picture of the stream on the definition page, the stream bed is wide enough to not be able to jump across and I assume the water levels reach full stream bed level at least part of the time. In that way, the example I provided is similar to a stream. Although Churn Creek has a rather wide bank, its flow is never constant and it hardly ever reaches peak flow rates. A lot of the time it can be crossed fairly easy. Although there some instances where it cant be. To me, a river never gets to the level where it can be safely crossed. Even at its lowest point. There is always a constantly high water flow rate. Plus the local government classifies Churn Creek as a creek and hydrology isn't really my area expertise. So I don't know. That's what I went with. 

Although I admit it could be more of a labeling issue in some instances, It think that is secondary to the merits of displaying the names of notable streams at higher levels or not. To me it would help people to be able to geographically orient themselves better at z14 in places with lots of waterways and add some context where there isn't many other local features to get their baring from. Plus I know there are people who use the map for planning hikes etc and it could possibly be helpful to them also.
